core/src/main/java/bio/terra/pearl/core/model/participant/Enrollee.java

```diff
@@ -39,5 +37,7 @@ public class Enrollee extends BaseEntity {
     @Builder.Default
     private Set<ParticipantTask> participantTasks = new HashSet<>();
     @Builder.Default
-    private Set<KitRequest> kitRequests = new HashSet<>();
+    private List<ParticipantNote> participantNotes = new ArrayList<>();
```

Comment:
I originally made most collections Sets because Sets play much nicer with Hibernate.  I think we should start moving these Collections on our models to Lists since that's what our DB calls return, and will allow us to track and return sorted collections.

Reply:
Yeah, this is always an ORM balancing act. `Set` makes sense in the object model to represent bags of distinct objects (no duplicates). However, database queries return rows in an order, even if that order is undefined (or even non-deterministic across executions of the same query). Therefore, using `List` is accurate, but sometimes implies too much; yes, the items are in an order, but that order may not have any significance. I try to avoid using `List` when the order is not deliberate and significant.

If we're going to use `List`, I think we should make an effort to fetch the elements in a consistent order and document it.

Reply:
Agree about consistent ordering.  What I also like about List is that it reminds us that we always send them to the client as a list, and so they're going to be put in *an* order, so we might as well make it one we control